xbmc/platform/posix/filesystem/PosixFile.h

```diff
@@ -10,6 +10,10 @@
 
 #include "filesystem/IFile.h"
 
+#if defined(STATX_TYPE) && !defined(ANDROID) // use statx if available to get file birth date
```

Comment:
What's the issue with android?

Reply:
I can't honestly remember now apart from the fact that Jenkins went red for all the Android builds.  I can do a test build just on Android to clarify what it was.